client/src/components/splitList.jsx

```diff
@@ -67,9 +84,16 @@ const SplitList = (props) => {
   return (
   <ul>
     {props.guests.map((guest, i) =>
-      <IndividualOwes key = {i} firstName = {guest.guestName} paymentOwed = {paymentsOwed[guest.guestName]} />
+      <IndividualOwes key = {i} guestName = {guest.guestName} paymentOwed = {paymentsOwed[guest.guestName]} />
     )}
-    Total: ${billWithTip.toFixed(2)}
+    <div>
+      <hr />
+      </div> Preliminary Total: ${billTotalWithoutTipOrTax.toFixed(2)} <div>
```

Comment:
we have some reversed divs here.

```suggestion
      <div> Preliminary Total: ${billTotalWithoutTipOrTax.toFixed(2)} </div>
```